src/storyblok-auth-api/handle-requests/handleSignIn/handleSignInRequest.ts

```diff
@@ -14,7 +14,7 @@ export const handleSignInRequest: HandleAuthRequest<{
 
   try {
     // TODO get rid of dummy spaceId
-    const redirectTo = openidClient(params).authorizationUrl({
+    const redirectTo = openidClient(params, 0).authorizationUrl({
```

Comment:
we don't need a proper spaceId here?

Reply:
No, because we only use the openId client to find out the URL to which we should redirect to, and this endpoint is shared between all spaces. If we needed a `spaceId` here, we'd be out of luck, because this code is executed before we find out the `spaceId`.

Reply:
Ah okay.


https://github.com/storyblok/app-extension-auth/blob/b6e362c53155d163430bbefa4df8da471bb2c9c1/src/storyblok-auth-api/handle-requests/openidClient.ts#LL18C6-L18C6

This made me think somehow it still affects the return of this function call.

Then maybe we need a different comment instead of `get rid of dummy spaceId`.